srcpkgs/pwget/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'pwget'
+pkgname=pwget
+version=2.0
+revision=1
+build_style=go
+go_import_path="github.com/majewsky/pwget"
+hostmakedepends="git"
+short_desc="Single-binary stateless password manager"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/majewsky/pwget"
+changelog="https://github.com/majewsky/pwget/raw/master/CHANGELOG.md"
+distfiles="https://github.com/majewsky/pwget/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum=052f167455e09b1c37f5e7db399952a8e8cd69661d5fff8462f39de7db0c5bf8
+
+post_install() {
+	vlicense LICENSE
```

Comment:
GPL-3.0-or-later is a standard license. I don't think it has to be included in the binary package.